### PR TITLE
Analpa 2614 bendparameters

### PR DESCRIPTION
**Pull request by henriikkaha**
Changed infotext, added autocomplete for gdogids, made icons for parameters, changed components to mui

### CODE REVIEW

File: src/icons/BendRadiusIconSelected.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      // style="background-color: rgb(255, 255, 255);"
```
Comment (CeduDev) [inline src/icons/BendRadiusIconSelected.js]:
```suggestion
```

Comment (CeduDev) [reply src/icons/BendRadiusIconSelected.js]: Remove commented code

**Comment by CeduDev** [comment]
As can be seen in the image, the content in the button is not properly alligned to the center. This left side has more space to the than the right side (all three buttons). This is most likely due to the svg's have a viewbox of fixed size which is wider than the actual svg image. Not a critical thing but could be nice to fix if possible.
image

EDIT: After consideration, since the svg is imported from mui-icons, it's a bit clunky to change the viewbox -> no need to change

**Comment by CeduDev** [comment]
> As can be seen in the image, the content in the button is not properly alligned to the center. This left side has more space to the than the right side (all three buttons). This is most likely due to the svg's have a viewbox of fixed size which is wider than the actual svg image. Not a critical thing but could be nice to fix if possible. image

EDIT: Fixed

Additionally, the buttons could be given a small margin as they look very cramped between the table and the tab

**Comment by CeduDev** [comment]
I'd recommed setting a top and bottom margin around the blue button. The size could be the same as between the grey submit button and the white area beneath it.
image

EDIT: Fixed

**Review by CeduDev — CHANGES_REQUESTED** [review]
Looks good, love the images and svgs used! The given suggestions could be addressed/applied, otherwise it's good

File: src/icons/BendRadiusIconSelected.js (hunk 1 above)
Comment (CeduDev) [reply src/icons/BendRadiusIconSelected.js]: fixed

**Review by CeduDev — APPROVED** [review]

**Review by adikele — APPROVED** [review]

**Pull request merged**